setup.py

```diff
@@ -49,16 +49,15 @@
     author_email='pipermerriam@gmail.com',
     url='https://github.com/ethereum/eth-tester',
     include_package_data=True,
-    install_requires=[
-        "toolz>0.8.2,<1;implementation_name=='pypy'",
-        "cytoolz>=0.8.2,<1.0.0;implementation_name=='cpython'",
+    install_requires=[    
+        "eth-abi>=1.0.0,<2",
+        "eth-keys>=0.2.0-beta.3,<0.3.0",
         "eth-utils>=1.1.1,<2.0.0",
         "rlp>=0.6.0,<2.0.0",
         "semantic_version>=2.6.0,<3.0.0",
-        "eth-keys>=0.2.0-beta.3,<0.3.0",
-        "eth-abi>=1.0.0-beta.1,<2",
     ],
     extras_require=extras_require,
+    python_requires='>=3.5,!=3.5.2,<4',
```

Comment:
Restricted 3.5.2 because of the `python/typing` issue that's popped up in a few other repos.